### PR TITLE
[imagenet example]use ```lr_scheduler``` instead of ```adjust_learning_rate```

### PR DESCRIPTION
**Pull request by jxgu1016**
PyTorch already have this function ```lr_scheduler``` to modify the learning rate,  so why we still use user-customized function ```adjust_learning_rate```?

### CODE REVIEW

**Comment by soumith** [comment]
this is done for coherence of example, i.e. to not introduce all concepts at once.
